server/serverAPI/model/user/requestSchema/UserLoginSchema.ts

```diff
@@ -0,0 +1,34 @@
+import { IsNotEmpty, IsString, validate } from "class-validator";
```

Comment:
Can some of the shared logic for this and UserRegistrationSchema be extracted?

Reply:
UserRegistrationSchema already shares logic from UserSchema

Reply:
I will probably need to redo partly inheritance at some later date...